cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -100,22 +100,28 @@ func NewOVFImporter(params *ovfimportparams.OVFImportParams) (*OVFImporter, erro
 		return nil, err
 	}
 	tarGcsExtractor := storageutils.NewTarGcsExtractor(ctx, storageClient, logger)
-	buildID := os.Getenv("BUILD_ID")
-
-	if buildID == "" {
-		buildID = pathutils.RandString(5)
-	}
 	workingDirOVFImportWorkflow := toWorkingDir(getImportWorkflowPath(params), params)
 	bic := &storageutils.BucketIteratorCreator{}
 
 	ovfImporter := &OVFImporter{ctx: ctx, storageClient: storageClient, computeClient: computeClient,
-		tarGcsExtractor: tarGcsExtractor, workflowPath: workingDirOVFImportWorkflow, BuildID: buildID,
+		tarGcsExtractor: tarGcsExtractor, workflowPath: workingDirOVFImportWorkflow, BuildID: getBuildID(params),
 		ovfDescriptorLoader: ovfutils.NewOvfDescriptorLoader(storageClient),
 		mgce:                &computeutils.MetadataGCE{}, bucketIteratorCreator: bic, Logger: logger,
 		zoneValidator: &computeutils.ZoneValidator{ComputeClient: computeClient}, params: params}
 	return ovfImporter, nil
 }
 
+func getBuildID(params *ovfimportparams.OVFImportParams) string {
+	if params != nil && params.BuildID != "" {
+		return params.BuildID
+	}
+	buildID := os.Getenv("BUILD_ID")
```

Comment:
can we just simply attach a rand string for whatever build id?

Reply:
I guess we could, I just didn't want to change the behavior in case anyone has relied on the fact build ID is used for temporary paths.